README.md

```diff
@@ -82,7 +82,7 @@ This guide will always assume you are starting with the latest TypeScript versio
 
 ## React + TypeScript Starter Kits
 
-1. [Create React App v2.1+ with Typescript](https://facebook.github.io/create-react-app/docs/adding-typescript): `npm create react-app my-new-react-typescript-app --typescript`
+1. [Create React App v2.1+ with Typescript](https://facebook.github.io/create-react-app/docs/adding-typescript): `npx create-react-app my-new-react-typescript-app --typescript`
```

Comment:
Changed to `npx` because every user may not have `create-react-app` installed globally but chances are they have `npx` since it is installed with `npm@5.2.0`.

Also, added a missing `-` in `create-react-app`.

Reply:
fyi `npm create` does work :) but ok